Cargo.lock

```diff
@@ -117,7 +171,7 @@ dependencies = [
  "futures-core",
  "memchr",
  "pin-project-lite",
- "tokio",
+ "tokio 1.25.0",
```

Comment:
Ugh looks like there are two version of tokio being pulled in, should run `cargo update` at least for tokio

Reply:
Good catch, this problem is already in sendme: https://github.com/n0-computer/sendme/pull/137